Voyage/Voyage.WebAPI/Program.cs

```diff
@@ -19,10 +22,24 @@
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+builder.Services.AddAuthentication(IdentityServerAuthenticationDefaults.AuthenticationScheme)
+      .AddIdentityServerAuthentication(IdentityServerAuthenticationDefaults.AuthenticationScheme, options =>
+      {
+          options.Authority = "https://localhost:5084";
+          options.ApiName = "Voyage";
+          options.RequireHttpsMetadata = false;
+      });
+
 builder.Services
     .AddDataAccess(options => options.BindConfiguration((nameof(DatabaseConfigs))))
     .AddBusinessLogic();
 
+var databaseConfigs = builder.Configuration
+    .GetSection(nameof(DatabaseConfigs))
+    .Get<DatabaseConfigs>();
+
+builder.Services.AddIdentityService(databaseConfigs);
+
 builder.Services.ConfigureOptions<ConfigureSwaggerOptions>();
```

Comment:
please, check application running, cause I have some issues with swagger and it's schemaId's